config/samples/sw.kogito_v08_kogitoserverlessbuild.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: sw.kogito.kie.org/v08
```

Comment:
please update filename

Reply:
Renamed and updated the reference into the kustomization file